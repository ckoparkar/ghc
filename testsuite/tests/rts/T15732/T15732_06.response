blah +RTS -G4 -RTS
