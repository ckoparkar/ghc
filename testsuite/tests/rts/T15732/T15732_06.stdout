args: ["a","b","c","@T15732_06.response"]
argsR: ["a","b","c","blah","+RTS","-G4","-RTS"]
oldGenFactor: 3.0
generations: 2
